src/snowflake/connector/auth/webbrowser.py

```diff
@@ -166,17 +166,17 @@ def prepare(
                 )
                 return
 
-            print(
+            logger.info(
                 "Initiating login request with your identity provider. A "
                 "browser window should have opened for you to complete the "
                 "login. If you can't see it, check existing browser windows, "
                 "or your OS settings. Press CTRL+C to abort and try again..."
             )
 
             logger.debug("step 2: open a browser")
-            print(f"Going to open: {sso_url} to authenticate...")
```

Comment:
We print it to stdout so when the program failed to open a browser, CLI users could manually open the link in browser, not sure if logging would work in this case.

Reply:
cc: @sfc-gh-mkeller

Reply:
This is at least a breaking change and should go through announcement. But I'm not too sure whether we'd want to change this in the first place. I think the opening message we wouldn't want to get rid off in the default case.
How about introducing a setting that switches to logging instead of printing, but we'd leave the default setting on printing? @invisiblethreat would that work for you?

Reply:
Yes, that would work. I'd actually propose that log is the default and a more verbose mode is available for troubleshooting, or this particular use case. Defaulting a library to STDOUT feels like the wrong thing to do, but I'm not privy to all the use cases 😄